runtime/kusama/src/lib.rs

```diff
@@ -955,13 +955,6 @@ impl pallet_proxy::Config for Runtime {
 	type AnnouncementDepositFactor = AnnouncementDepositFactor;
 }
 
-pub struct CustomOnRuntimeUpgrade;
-impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
-	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		0
-	}
-}
```

Comment:
this code wasn't use in executive 😮
We should really not keep dead code like this, either add it to executive, or like you did remove completly

Reply:
yeah crazy.